eyetrack_eeg/sub-001/eye/sub-001_task-HBNdata_events.tsv

```diff
@@ -0,0 +1,133 @@
+onset	duration	sample	value	sync_event
+0	n/a	0	n/a	0
+0.007462687	n/a	1	n/a	0
+0.014925373	8	2	0	0
+0.074626866	226	10	93	0
+1.76119403	46	236	4	1
+2.104477612	348	282	8	1
+4.701492537	76	630	4	1
+5.268656716	433	706	8	1
```

Comment:
@arnodelorme 
are those durations in seconds?

Reply:
yes, first column is in second. The second column is in samples for the EEG (EEG files) and samples for eye-tracking (eye-tracking file) - but at different sampling rates.

Reply:
duration column MUST be in seconds

https://bids-specification.readthedocs.io/en/latest/glossary.html#objects.columns.duration

samples will have to be dropped from the events.tsv if we are merging data with different sampling scheme: this was the old recommendation in the spec before the `samples` column got dropped altogether from the spec

Reply:
OK, I understand about dropping the "sample" column.